build/package/Dockerfile

```diff
@@ -0,0 +1,21 @@
+FROM brandoshmando/go:1.13.4-ticket-lock AS binary
+
+ADD . /build
```

Comment:
If we move this below `apk add` we can avoid rerunning that step whenever a code change occurs.